cmd/cloud.go

```diff
@@ -247,6 +276,14 @@ This will execute the test on the Load Impact cloud service. Use "k6 login cloud
 		var progressErr error
 		ticker := time.NewTicker(time.Millisecond * 2000)
 		shouldExitLoop := false
+		if showCloudLogs {
+			go func() {
+				// TODO replace with another context
+				if err := cloudConfig.StreamLogsToLogger(context.Background(), logger, refID, 0, 100); err != nil {
```

Comment:
Why `100`? Why have a limit at all, given that the cloud already does a ton of log rate-limiting :confused: Having holes in the middle of your logs, potentially caused by 2 completely different places seems... unwise :confused: Can't this drop the cloud message for dropped log lines, for example?

Reply:
this limits how many logs will be in each message. 

There could still be holes because we couldn't get the logs fast enough. I have also removed this as I don't think it matters all that much, the dropping of the logs (because they can't be pushed to us fast enough) didn't seem to be impacted by it either way when I tested earlier, it mostly had relevance on how fast old logs were returned (where there were problems as well)